docs/Emerging-Issues.md

```diff
@@ -11,7 +11,7 @@ This page lists emerging issues for Exchange On-Premises deployments, possible r
 
 **Issue** |**Possible reason**| **Workaround/Solution**
 -|-|-
-After installing [March 2022 Security Update For Exchange Server 2013, 2016, 2019](https://techcommunity.microsoft.com/t5/exchange-team-blog/released-march-2022-exchange-server-security-updates/ba-p/3247586), the Microsoft Exchange Service Host service may crash repeatedly with Event ID 7031 in system log and Event ID 4999 in application log. <BR><BR>  Event ID 4999 <BR> Watson report about to be sent for process id: 4564, with parameters: E12IIS, c-RTL-AMD64, 15.01.2375.024, M.Exchange.ServiceHost, M.Exchange.Diagnostics, M.E.D.ChainedSerializationBinder.LoadType, M.E.Diagnostics.BlockedDeserializeTypeException, c0e9-dumptidset, 15.01.2375.024.| The issue can occur if there are any expired certificates present on or any certificates nearing expiry on the server| •	Replace any expired certificate on the system <BR> •	Renew any certificate that expires in <= 30 days
+After installing [March 2022 Security Update For Exchange Server 2013, 2016, 2019](https://techcommunity.microsoft.com/t5/exchange-team-blog/released-march-2022-exchange-server-security-updates/ba-p/3247586), the Microsoft Exchange Service Host service may crash repeatedly with Event ID 7031 in system log and Event ID 4999 in application log. <BR><BR>  Event ID 4999 <BR> Watson report about to be sent for process id: 4564, with parameters: E12IIS, c-RTL-AMD64, 15.01.2375.024, M.Exchange.ServiceHost, M.Exchange.Diagnostics, M.E.D.ChainedSerializationBinder.LoadType, M.E.Diagnostics.BlockedDeserializeTypeException, c0e9-dumptidset, 15.01.2375.024.| The issue can occur if there are any expired certificates present on or any certificates nearing expiry on the server| Use following steps to remove the expiry certificate notifications: <BR> 1)	The user will need to have full access to arbitration mailbox <BR> a.	Get-Mailbox -Arbitration "SystemMailbox{e0dc1c29-89c3-4034-b678-e6c29d823ed9}" <BR> 2) Add-MailboxPermission -User administrator -AccessRights FullAccess <BR> 3)	Run [this Script](https://aka.ms/RemoveCertExpiryNotifications) from Exchange Management Shell login as the user with full permission assigned to arbitration mailbox <BR> 4)	Confirm all the messages are deleted <BR> 5)	Start the service MSExchangeServiceHost and confirm that it is not Crashing <BR> 6)	Revoke the full Access permission for the user <BR> a.	Get-Mailbox -Arbitration "SystemMailbox{e0dc1c29-89c3-4034-b678-e6c29d823ed9}" <BR> 7) Remove-MailboxPermission -User administrator -AccessRights FullAccess 
```

Comment:
The adding and the removing the permissions formatting isn't correct. It looks like this:

```
a. Get-Mailbox -Arbitration "SystemMailbox{e0dc1c29-89c3-4034-b678-e6c29d823ed9}"
2) Add-MailboxPermission -User administrator -AccessRights FullAccess
```

Reply:
Fixed it. Had to look up how to escape a pipe in a table.